.pre-commit-hooks.yaml

```
@@ -5,3 +5,4 @@
     language: python
     files: "dependencies.yaml"
     pass_filenames: false
+    args: [-Wall, -Werror]
```

```suggestion
    args: [--warn-all, --strict]
```

This should get changed now that the spelling for the options changed.